lib/util.js

```diff
@@ -60,40 +66,7 @@ export function privateKeyDerEncode({privateKeyBytes, seedBytes}) {
     // extract the first 32 bytes of the 64 byte private key representation
     p = Buffer.from(privateKeyBytes.buffer, privateKeyBytes.byteOffset, 32);
```

Comment:
No need to get at the underlying ArrayBuffer, just use `slice`:

```suggestion
    p = privateKeyBytes.slice(0, 32);
```

(Please confirm I didn't mess that up :) ).